src/Cache/CachedTrait.php

```diff
@@ -33,26 +33,18 @@ private function getCached(string $cacheKey, callable $getValue)
 
         try {
             $cacheItem = $this->cacheItemPool->getItem($cacheKey);
-
-            if ($cacheItem->isHit()) {
-                return $this->localCache[$cacheKey] = $cacheItem->get();
-            }
         } catch (CacheException $e) {
-            //do nothing
+            return $this->localCache[$cacheKey] = $getValue();
         }
 
-        $value = $getValue();
-
-        if (!isset($cacheItem)) {
-            return $this->localCache[$cacheKey] = $value;
+        if ($cacheItem->isHit()) {
+            return $this->localCache[$cacheKey] = $cacheItem->get();
         }
 
-        try {
-            $cacheItem->set($value);
-            $this->cacheItemPool->save($cacheItem);
-        } catch (CacheException $e) {
```

Comment:
I was only copying exciting code in the previous pr so I am not sure of the original author's thoughts, but you are now keeping the exception handling above but removing this one. Should we be consistent?

Reply:
There is no chance of `CacheException` being thrown here. The original author (i.e. myself) was mistaken. :laughing:

Reply:
And we all copied without thinking twice :laughing: